### ISSUE TITLE
Add log level selector

### ISSUE THREAD

**Issue by Aijuh** [snu-quiqcl/qiwis#199]
### Feature to implement

Add an additional widget to select log level.
 

### Implementation plan

1. Inside LoggerApp class, add a new combo widget(name :levelHandler) to select log level (DEBUG, INFO, WARNING, ERROR, CRITICAL)
2. Connect a new function(name : fixLevel) to combo widget inside the class LoggerApp
3. Inside the function fixLevel, modify Level of LoggerApp's handler.

### Additional Info

To modify logger level of LoggerApp using handler, logger's level should be set as 'DEBUG'. If logger's level is 'INFO', by changing level of handler, we can block 'INFO' or 'WARNING' signal. But 'DEBUG' signal cannot reach to logger by changing handler's level.
In other words handler can block more serious level, but cannot allow less serious level to pass.

Currently we are using root logger as LoggerApp's logger and change its level as 'DEBUG'. If we cannot change root logger's level, implementation should be changed.
